aerospike_helpers/expressions/base.py

```diff
@@ -32,6 +32,8 @@
 from aerospike_helpers.expressions.resources import _ExprOp
 from aerospike_helpers.expressions.resources import ResultType
 from aerospike_helpers.expressions.resources import _Keys
+from aerospike_helpers.cdt_ctx import _cdt_ctx
+from abc import ABC
```

Comment:
Is ABC necessary here?  You aren't using any decorators or methods from ABC?
I have seen that ABC has some overhead implications: [Stack Overflow page](https://stackoverflow.com/questions/34846631/computational-cost-of-abc)